stat.c

```diff
@@ -482,9 +482,12 @@ static void show_ddir_status(struct group_run_stats *rs, struct thread_stat *ts,
 		display_lat("clat", min, max, mean, dev, out);
 	if (calc_lat(&ts->lat_stat[ddir], &min, &max, &mean, &dev))
 		display_lat(" lat", min, max, mean, dev, out);
+	if (ddir == DDIR_READ && calc_lat(&ts->clat_high_prio_stat, &min, &max, &mean, &dev))
```

Comment:
Not sure I follow why you gate all of these on DDIR_READ?

Reply:
Current Seagate implementation of handling the priority bit will only affect reads, and my understanding is WD's implementation is the same. There's no guarantee this will be true in the future, but I figured it is better to only track reads for now.

Reply:
That limitation on the device front should not be carried into fio. You're setting the priority bit regardless of data direction, you should print the stats similarly as well.

Reply:
I'll update it to also collect high/normal priority completion latencies separately for reads, writes and trims. I wonder if it would make sense to be able to set cmdprio_percentage separately for each data direction?